homeassistant/components/person/__init__.py

```diff
@@ -436,47 +456,10 @@ def from_yaml(cls, config: ConfigType) -> Self:
         person.editable = False
         return person
 
-    @property
-    def name(self) -> str:
-        """Return the name of the entity."""
-        return self._config[CONF_NAME]
-
-    @property
-    def entity_picture(self) -> str | None:
-        """Return entity picture."""
-        return self._config.get(CONF_PICTURE)
-
-    @property
-    def state(self) -> str | None:
-        """Return the state of the person."""
-        return self._state
-
-    @property
-    def extra_state_attributes(self) -> dict[str, Any]:
-        """Return the state attributes of the person."""
-        data: dict[str, Any] = {ATTR_EDITABLE: self.editable, ATTR_ID: self.unique_id}
-        if self._latitude is not None:
-            data[ATTR_LATITUDE] = self._latitude
-        if self._longitude is not None:
-            data[ATTR_LONGITUDE] = self._longitude
-        if self._gps_accuracy is not None:
-            data[ATTR_GPS_ACCURACY] = self._gps_accuracy
-        if self._source is not None:
-            data[ATTR_SOURCE] = self._source
-        if (user_id := self._config.get(CONF_USER_ID)) is not None:
-            data[ATTR_USER_ID] = user_id
-        data[ATTR_DEVICE_TRACKERS] = self.device_trackers
-        return data
-
-    @property
-    def unique_id(self) -> str:
-        """Return a unique ID for the person."""
-        return self._config[CONF_ID]
-
-    @property
+    @cached_property
     def device_trackers(self) -> list[str]:
```

Comment:
Why do we need a property for `device_trackers`? I'd try to just use a attribute.